client/app/billing/autoRenew/autorenew.service.js

```diff
@@ -77,6 +84,13 @@ export default class {
     }), {});
   }
 
+  getStatesTypes() {
```

Comment:
```suggestion
  getStateTypes() {
```
but this is nitpicking :)